src/Core/Framework/Store/Command/StoreLoginCommand.php

```diff
@@ -74,6 +70,21 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $password = $input->getOption('password');
         $user = $input->getOption('user');
 
+        if (!$password) {
+            $passwordQuestion = new Question('Enter password');
+            $passwordQuestion->setValidator(static function ($value): string {
+                if ($value === null || trim($value) === '') {
+                    throw new \RuntimeException('The password cannot be empty');
+                }
+
+                return $value;
```

Comment:
these lines are not covered

Reply:
Due to QuestionHelper it's throwing a MissingInputException because of a stdin error. Now I'm mobile and can't verify but I'm certain that RuntimeException it's catched by the max retries functionality.